migrations/002.do.characters-update.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE characters ADD CONSTRAINT characters_unique_name UNIQUE (player_id, name);
+ALTER TABLE characters ADD COLUMN charge INT NOT NULL DEFAULT 0;
```

Comment:
I think this charge goes here too? Has to be stored somewhere, so I just stuck it in.

Reply:
Yup, that looks right!